cypress_shared/pages/temporary-accommodation/manage/bedspaceSearch.ts

```diff
@@ -53,6 +53,6 @@ export default class BedspaceSearchPage extends Page {
   }
 
   clickBedspaceLink(room: Room) {
-    cy.get('h4').contains(room.name).parent().siblings().contains('View').click()
+    cy.get('td').contains(room.name).parent().find('td:contains("View")').click()
```

Comment:
Ah yes, I think I was trying alternatives because the test suite started reporting that there were no tests around this time. I wrongly assumed this latest change was to blame and forgot to put it back.